src/db.c

```diff
@@ -96,53 +113,17 @@ robj *lookupKey(redisDb *db, robj *key, int flags) {
  *
  *  LOOKUP_NONE (or zero): no special flags are passed.
  *  LOOKUP_NOTOUCH: don't alter the last access time of the key.
+ *  LOOKUP_NONOTIFY: don't trigger keyspace event on key miss.
+ *  LOOKUP_NOSTATS: don't increment key hits/misses couters.
+ *  LOOKUP_NOEXPIRE: don't check if the key has expired.
```

Comment:
LOOKUP_WRITE is not listed.

Reply:
Well, I don't think lookupKeyRead should be used with the WRITE flag... I'll move all this to the lookupKey top comment.